src/Users/v1.0/examples/Get-MgUserMemberOf.md

```diff
@@ -1,9 +1,17 @@
-### Example 1: Code snippet
+### Example 1: {{ Add title here }}
+```powershell
+ PS C:\> {{ Add code here }}
 
-```powershell
Import-Module Microsoft.Graph.Users
+{{ Add output here }}
+```
 
-Get-MgUserMemberOf -UserId $userId
-```
-This example shows how to use the Get-MgUserMemberOf Cmdlet.
-
To learn about permissions for this resource, see the [permissions reference](/graph/permissions-reference).
-
+{{ Add description here }}
```

Comment:
@timayabi2020, it looks like some code snippets that were previously available are being removed by this PR. Could this be due to the snippets change made in DevX API?